### PR TITLE
Added work-dir and moved hidden files

### LINKED ISSUE #13: permission error when cloning repositories via docker run
When trying to clone a new repository on my local system using this image (with a volume set up) I get permission issues.

This is the command I'm trying to run:

```shell
docker run -it -v ${pwd}:/home/nonroot --rm distroless.dev/git clone https://github.com/distroless/static.git
```

The response I get:

```
fatal: could not create work tree dir 'static': Permission denied
```
Maybe we need to update `apko.yml` to set up `/home/nonroot` for the `nonroot` user?

### PR DESCRIPTION
**Pull request by amouat**
This should fix #13 

Also changed default user. 

Signed-off-by: Adrian Mouat <adrian@chainguard.dev>

### CODE REVIEW

File: .github/workflows/pre-submit.yaml [hunk 1]
```diff
@@ -26,12 +26,12 @@ jobs:
...
     - uses: distroless/actions/apko-build@main
       with:
-        config: .apko.yaml
+        config: apko.yaml
         tag: template:${{ steps.snapshot-date.outputs.date }}
 
     - uses: distroless/actions/apko-build@main
       with:
-        config: .apko-root.yaml
+        config: apko-root.yaml
```
Comment (cpanato) [inline .github/workflows/pre-submit.yaml:34]: the files have the `.` in front https://github.com/distroless/git/blob/main/.apko-root.yaml
this is correct?

Comment (amouat) [reply .github/workflows/pre-submit.yaml:34]: The files should be moved as well. It's a bit hard to tell in GitHub...

Comment (amouat) [reply .github/workflows/pre-submit.yaml:34]: Yes, I moved the files.

**Review by cpanato — APPROVED** [review]

File: .github/workflows/pre-submit.yaml (hunk 1 above)
Comment (cpanato) [reply .github/workflows/pre-submit.yaml:34]: ah i see that now

**Pull request merged**